auctionApp/src/main/java/com/vmware/weathervane/auction/data/imageStore/ImageStoreFacadeCassandraImpl.java

```diff
@@ -255,33 +255,44 @@ public void addImages(List<ImageInfo> imageInfos, List<BufferedImage> fullImages
 	public byte[] retrieveImage(UUID imageHandle, ImageSize size) throws NoSuchImageException,
 			IOException {
 		logger.info("retrieveImage imageHandle = " + imageHandle + ", imageSize = " + size);
+		List<ImageThumbnail> thumbs = imageThumbnailRepository.findByKeyImageId(imageHandle);
+		List<ImagePreview> previews = imagePreviewRepository.findByKeyImageId(imageHandle);
+		List<ImageFull> fulls = imageFullRepository.findByKeyImageId(imageHandle);
 		switch (size) {
```

Comment:
Will retrieving all 3 images vs just 1 depending on the case have a performance impact?

Reply:
Yes.  Good catch.  I will change this.

Reply:
fixed